src/Test/L0/Listener/SelfUpdaterL0.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if !(OS_OSX && ARM64)
+﻿#if !((OS_OSX || OS_WINDOWS) && ARM64)
```

Comment:
```suggestion
﻿#if !(OS_WINDOWS && ARM64)
```

I think we were commenting out `OS_OSX and ARM64` in the initial release since it can't do any upgrade testing.